nimble/host/services/asha/include/services/asha/ble_svc_asha.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef H_BLE_SRV_ASHA_
+#define H_BLE_SRV_ASHA_
+
+#include <stdbool.h>
+#include "nimble/ble.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define CONTROL_POINT_OP_START        0x01
+#define CONTROL_POINT_OP_STOP         0x02
+#define STATUS_OK                     0x00
+#define UNKNOWN_COMMAND               0x01
+#define ILLIGAL_PARAMETERS            0x02
+#define LEFT                          0X02
+#define RIGHT                         0X03
+#define DEVICE_SIDE                   RIGHT 
+#define READ_ONLY_BUFFER_SIZE         17
+/* Future use. */
+#define  CODEC_G722_16KHZ             0x01
+#define  CODEC_G722_24KHZ             0x02
+
+/********* Hearing aid service. ***********************/
+#define BLE_SVC_ASHA_UUID16     0xFDF0
+
+/* ASHA Characteristics UUIDS. */
+/*6333651e-c481-4a3e-9169-7c902aad37bb*/
+#define BLE_SVC_ASHA_CHR_UUID128_DEVICE_PROP \
+           0xbb, 0x37, 0xad, 0x2a, 0x90, 0x7c, 0x69, 0x91\
+           ,0x3e,  0x4a, 0x81, 0xc4, 0x1e, 0x65, 0x33, 0x63
```

Comment:
do we need to expose uuids in header file? I guess those should be `ble_uuid128_t` defined in `.c` - see `bleuart.c`